portal-api/src/v1/users/routes.js

```diff
@@ -356,7 +248,7 @@ module.exports.resetPasswordWithToken = async (req, res, next) => {
     });
   }
 
-  const errors = applyUpdateRules(user, {
+  const errors = await applyUpdateRules(user, {
     password,
     passwordConfirm,
   });
```

Comment:
I do not understand _why_ we would ever have a `currentPassword` here considering whe have a reset token. But just in case -- as the note implies its needed

Reply:
@AlexBramhill Which note? It looks `currentPassword` is just being added here?